tests/conftest.py

```diff
@@ -233,7 +513,7 @@ def create_client(request) -> weave_init.InitializedClient:
 
 
 @pytest.fixture()
-def client(request) -> Generator[weave_client.WeaveClient, None, None]:
+def client(request):
```

Comment:
why get rid of the typing?

Reply:
I don't think it's helpful in this context